RedHatInsights/insights-inventory-frontend
# fix(ESSNTL-5136): Check RBAC for null group id when a host is not in any group

Implements https://issues.redhat.com/browse/ESSNTL-5136.

Must be merged after https://github.com/RedHatInsights/frontend-components/pull/1886 is released.

At /inventory/%id, users must have Delete and Edit actions enabled for hosts that are not in any group if the users have write permissions with group.id set to `null`.

- [x] Write cy tests

## How to test

1. Limit your inventory:hosts:write and inventory:hosts:read permission to only ungroupped hosts by setting `group.id` to null (see more details in the attached Jira)
2. Navigate to any ungroupped host and make sure you can delete it or edit its details.
